Solutions/VaronisSaaS/Data Connectors/azuredeploy.json

```diff
@@ -0,0 +1,257 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.24.24.22086",
+      "templateHash": "1576663599375367683"
+    }
+  },
+  "parameters": {
+    "functionAppName": {
+      "type": "string",
+      "defaultValue": "[format('VaronisSaaS-{0}', uniqueString(resourceGroup().id))]",
```

Comment:
Will it be okay if we let user add the custom Function app name, 
Also the zip and function app is missing with python libraries. Please add those as well.
You can refer this Data connector for more clarification [Solutions/MimecastTIRegional/Data Connectors](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/MimecastTIRegional/Data%20Connectors)

Reply:
Hi @v-prasadboke,
Currently user can change function name if he wants. It's just a default value.
Also we use .net as a platform for azure function. You can find source code and zip package in "Data Connectors" folder.

![image](https://github.com/Azure/Azure-Sentinel/assets/99330808/8768a444-c0e2-49f2-91ba-73a4d202bf50)
![image](https://github.com/Azure/Azure-Sentinel/assets/99330808/3f1579d6-a1d2-4d99-ba67-b3616aae3010)